src/components/time-entries/TimeEntries.tsx

```diff
@@ -17,41 +19,70 @@ export const TimeEntries = ({ isModalActive, handleModal }: Types.AtBuildProps)
     deleteTimeEntries(entry);
   };
 
+  const sortOptions = ["ascending", "descending"];
+
+  const handleSort = () => {
+    const sort = timeEntries.sort((a, b) => {
+      if (a.startTime < b.startTime) {
+        return sortOrder === "ascending" ? 1 : -1;
+      }
+      if (a.startTime > b.startTime) {
+        return sortOrder === "ascending" ? -1 : 1;
+      }
+      return 0;
+    });
+
+    setSortedTimeEntries(sort);
+  };
+
+  useEffect(() => {
+    handleSort();
+  }, [sortOrder]);
+
   return (
     <>
       <Styled.Main>
-        {timeEntries
-          ?.sort((a, b) => new Date(b.startTime).valueOf() - new Date(a.startTime).valueOf())
-          .map((timeEntry, i, arr) => {
-            const currentDate = new Date(timeEntry.startTime).toLocaleDateString("en-EN", {
-              weekday: "long",
-              day: "numeric",
-              month: "numeric",
-            });
+        <Styled.Container>
+          <Styled.Label>Sort by Date:</Styled.Label>
+          <Styled.Select value={sortOrder} onChange={(e) => setSortOrder(e.target.value)}>
+            <option value="">--Select option--</option>
```

Comment:
Add `disabled` here, value should be either ascending or descending and cannot be empty.